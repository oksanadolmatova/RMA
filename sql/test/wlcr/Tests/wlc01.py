from __future__ import print_function

try:
    from MonetDBtesting import process
except ImportError:
    import process
import os, sys

dbfarm = os.getenv('GDK_DBFARM')
tstdb = os.getenv('TSTDB')

if not tstdb or not dbfarm:
    print('No TSTDB or GDK_DBFARM in environment')
    sys.exit(1)

#clean up first
dbname = tstdb
dbnameclone = tstdb + '-clone'

s = process.server(dbname = dbname, stdin = process.PIPE, stdout = process.PIPE, stderr = process.PIPE)

c = process.client('sql', dbname = dbname, stdin = process.PIPE, stdout = process.PIPE, stderr = process.PIPE)

cout, cerr = c.communicate('''\
call masterbeat(0);
call master();
create table tmp(i int, s string);
insert into tmp values(1,'hello'), (2,'world');
select * from tmp;
''')

sout, serr = s.communicate()

sys.stdout.write(sout)
sys.stdout.write(cout)
sys.stderr.write(serr)
sys.stderr.write(cerr)
