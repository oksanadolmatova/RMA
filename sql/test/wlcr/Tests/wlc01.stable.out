stdout of test 'wlc01` in directory 'sql/test/wlcr` itself:


# 21:28:50 >  
# 21:28:50 >  "/usr/bin/python2" "wlc01.py" "wlc01"
# 21:28:50 >  

# MonetDB 5 server v11.26.0
# This is an unreleased version
# Serving database 'mTests_sql_test_wlcr', using 8 threads
# Compiled for x86_64-unknown-linux-gnu/64bit with 128bit integers
# Found 15.589 GiB available main-memory.
# Copyright (c) 1993-July 2008 CWI.
# Copyright (c) August 2008-2017 MonetDB B.V., all rights reserved
# Visit http://www.monetdb.org/ for further information
# Listening for connection requests on mapi:monetdb://vienna.da.cwi.nl:31055/
# Listening for UNIX domain connection requests on mapi:monetdb:///var/tmp/mtest-7935/.s.monetdb.31055
# MonetDB/GIS module loaded
# MonetDB/SQL module loaded
# SQL catalog created, loading sql scripts once
# loading sql script: 09_like.sql
# loading sql script: 10_math.sql
# loading sql script: 11_times.sql
# loading sql script: 12_url.sql
# loading sql script: 13_date.sql
# loading sql script: 14_inet.sql
# loading sql script: 15_querylog.sql
# loading sql script: 16_tracelog.sql
# loading sql script: 17_temporal.sql
# loading sql script: 18_index.sql
# loading sql script: 20_vacuum.sql
# loading sql script: 21_dependency_functions.sql
# loading sql script: 22_clients.sql
# loading sql script: 23_skyserver.sql
# loading sql script: 25_debug.sql
# loading sql script: 26_sysmon.sql
# loading sql script: 27_rejects.sql
# loading sql script: 39_analytics.sql
# loading sql script: 39_analytics_hge.sql
# loading sql script: 40_geom.sql
# loading sql script: 40_json.sql
# loading sql script: 40_json_hge.sql
# loading sql script: 41_md5sum.sql
# loading sql script: 45_uuid.sql
# loading sql script: 46_gsl.sql
# loading sql script: 46_profiler.sql
# loading sql script: 51_sys_schema_extension.sql
# loading sql script: 60_wlcr.sql
# loading sql script: 72_fits.sql
# loading sql script: 74_netcdf.sql
# loading sql script: 75_lidar.sql
# loading sql script: 75_shp.sql
# loading sql script: 75_storagemodel.sql
# loading sql script: 80_statistics.sql
# loading sql script: 80_udf.sql
# loading sql script: 80_udf_hge.sql
# loading sql script: 85_bam.sql
# loading sql script: 90_generator.sql
# loading sql script: 90_generator_hge.sql
# loading sql script: 99_system.sql
#create table tmp(i int, s string);
#insert into tmp values(1,'hello'), (2,'world');
[ 2	]
#select * from tmp;
% sys.tmp,	sys.tmp # table_name
% i,	s # name
% int,	clob # type
% 1,	5 # length
[ 1,	"hello"	]
[ 2,	"world"	]

# 21:28:51 >  
# 21:28:51 >  "Done."
# 21:28:51 >  

